api/build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id 'java'
+    id 'war'
+}
+
+group 'com.epam.esm'
+version '1.0'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation project(':core')
+
+    compile group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.7.0'
```

Comment:
Could you please separate dependencies related to configuration `compile` and `testCompile`?
at first, all dependencies with `compile`, empty line, then all dependencies with `testCompile`
Such approach increases readability - easy to understand which dependencies for application and which ones only for tests